sympy/solvers/tests/test_ode.py

```diff
@@ -2552,3 +2552,14 @@ def test_issue_7093():
     sol = Eq(f(x), C1 - 2*x*sqrt(x**3)/5)
     eq = Derivative(f(x), x)**2 - x**3
     assert dsolve(eq) == sol and checkodesol(eq, sol) == (True, 0)
+
+
+def test_dsolve_linsystem_symbol():
+    x, y = symbols('x, y', function=True)
+    t = Symbol('t')
+    eps = symbols('epsilon', positive=True)
+
+    eq1 = (Eq(diff(x(t), t), -eps*y(t)), Eq(diff(y(t), t), eps*x(t)))
+    sol1 = [Eq(x(t), -eps*(C1*sin(eps*t) + C2*cos(eps*t))), 
+             Eq(y(t), C1*eps*cos(eps*t) - C2*eps*sin(eps*t))]
+    assert dsolve(eq1) == sol1
```

Comment:
Is there something wrong with the whitespace at the end of this file?  See test failure.